sys-apps/edac-utils/edac-utils-0.18-r1.ebuild

```diff
@@ -0,0 +1,75 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit autotools flag-o-matic toolchain-funcs
+
+DESCRIPTION="Userspace helper for Linux kernel EDAC drivers"
+HOMEPAGE="https://github.com/grondo/edac-utils"
+SRC_URI="https://github.com/grondo/${PN}/archive/${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="GPL-2"
+SLOT="0"
+KEYWORDS="~amd64"
+IUSE="debug"
+
+DEPEND="sys-fs/sysfsutils"
+RDEPEND="${DEPEND}
+	sys-apps/dmidecode"
+
+src_prepare() {
+	default
+
+	# <sys-devel/gcc-10, <sys-devel/binutils-2.34 do not support LDPT_ADD_SYMBOLS_V2
+	# https://sourceware.org/bugzilla/show_bug.cgi?id=25355
+	# We use Portage's build environment to determine which GCC and binutils version are
+	# active. If either fail to meet the minimum version, -flto* is stripped from FLAGS
+	local gcc_vers=$(gcc-version)
+	local ld_vers=$($(tc-getLD) -v | sed 's/.*\(2\.[0-9][0-9]\).*/\1/')
```

Comment:
Isn't this check already done in the `if` construct below? This sed seems rather redundant.